### PR TITLE
Introduce `SwiftLint` and `.swiftformat`

### PR DESCRIPTION
**Pull request by CanglongCl**
Most of changes are made by `swiftformat` which do not influence functions of the code.

### CODE REVIEW

File: .swiftformat [hunk 1]
```diff
@@ -1,4 +1,108 @@
-# https://github.com/nicklockwood/SwiftFormat/blob/main/Rules.md
+# SwiftFormat config compliant with Google Swift Guideline
+# https://google.github.io/swift/#control-flow-statements
 
-# file options
---exclude Pods
\ No newline at end of file
+# Specify version used in a project
+
+--swiftversion 5.7
```
Comment (tisfeng) [inline .swiftformat:6]: Can we use swift version 5.9 ?

Comment (tisfeng) [reply .swiftformat:6]: I have added a `.swift-version` file  to specify swift version in 41bcc869 , if use `--swiftversion`, can we remove that file?

Comment (CanglongCl) [reply .swiftformat]: This file also contains other rules which is consistent with swiftlint configuration. You may edit rules in this file but we'd better not to remove it.

Comment (tisfeng) [reply .swiftformat]: ok

**Review by tisfeng — DISMISSED** [review]
Looks good.

**Review by phlpsong — CHANGES_REQUESTED** [review]

File: .swiftlint.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  warning: 900
+  error: 1600
+type_name:
+  min_length: 3
+  max_length:
+    warning: 50
+    error: 50
+identifier_name:
+  min_length: 3
+  excluded: # excluded via string array
```
[the remark this answers is not in the document]
Comment (CanglongCl) [reply .swiftlint.yml:51]: No. Functions like 
```swift
override func translate(
    _ text: [String],
    from: Language,
    to: Language,
    completion: @escaping (EZQueryResult, Error?) -> ()
)
```
should be written as like 
```swift
override func translate(
    _ text: [String],
    from sourceLanguage: Language,
    to destinationLanguage: Language,
    completion: @escaping (EZQueryResult, Error?) -> ()
)
```
which is a more clear identifier.

[the remark this answers is not in the document]
Comment (phlpsong) [reply .swiftlint.yml:51]: ok

File: Easydict/Feature/Service/Ali/AliService.swift [hunk 3]
```diff
@@ -37,13 +25,8 @@ class AliService: QueryService {
...
     }
 
-    override func hasPrivateAPIKey() -> Bool {
-        let id = Defaults[.aliAccessKeyId] ?? ""
-        let secret = Defaults[.aliAccessKeySecret] ?? ""
-        return !id.isEmpty && !secret.isEmpty
-    }
-
     override public func supportLanguagesDictionary() -> MMOrderedDictionary<AnyObject, AnyObject> {
+        // swiftlint:disable:next todo
```
Comment (phlpsong) [inline Easydict/Feature/Service/Ali/AliService.swift:29]:
```suggestion
```

Comment (CanglongCl) [reply Easydict/Feature/Service/Ali/AliService.swift:29]: @tisfeng How do you think of the `todo` rule? Maybe leaving it is a better choice?

Comment (phlpsong) [reply Easydict/Feature/Service/Ali/AliService.swift:29]: I believe it will not trigger a warning after removing this line.

**Review by phlpsong — APPROVED** [review]

**Review by tisfeng — APPROVED** [review]

**Pull request merged**